windows/README.md

```diff
@@ -37,4 +37,19 @@ Get-Service -Name "SF*" | foreach {sc.exe config $_.Name start= disabled}
 
 ```
 Get-Service -Name "SF*" | foreach {sc.exe start $_.Name}
+```
+
+
+# Windows Command Line Scripts
+
+### Add extra SmartFace service
+
+#### Add extra Camera service
+```
+SC CREATE "SFCam11" binpath= "C:\Program Files\Innovatrics\SmartFace\SmartFace.Camera.exe  --serviceName SFCam11" DisplayName= "SmartFace Camera 11" start= delayed-auto
```

Comment:
There is no harm in having this here, but this is exactly the logic that is already "implemented" in the SF binaries package with a simple install .bat script.

Also maybe we should consider adding some kind of a SF windows service management cli which would perform these tasks